File: fix_hrefs.py
import re

with open('C:/Users/4513/.openclaw/workspace/jiugong-bagua/src/app/page.tsx', 'r', encoding='utf-8') as f:
    c = f.read()

# Map emojis to routes
replacements = [
    ('🧭', '/fengshui'),
    ('⚖️', '/chenggu'),
    ('🐉', '/shengxiao'),
    ('♈', '/xingzuo'),
    ('🌀', '/qimen'),
    ('🌸', '/meihua'),
    ('🏮', '/lingqian'),
]

for emoji, route in replacements:
    old = f"emoji: '{emoji}', href: '#'"
    new = f"emoji: '{emoji}', href: '{route}'"
    c = c.replace(old, new)

with open('C:/Users/4513/.openclaw/workspace/jiugong-bagua/src/app/page.tsx', 'w', encoding='utf-8') as f:
    f.write(c)

remaining = c.count("href: '#'")
print(f'Done. {remaining} # hrefs remaining')

File: fix_hrefs.js
const fs = require('fs');
let c = fs.readFileSync('C:/Users/4513/.openclaw/workspace/jiugong-bagua/src/app/page.tsx', 'utf8');
const reps = [
  ['\u{1F9ED}', '/fengshui'],
  ['\u2696\uFE0F', '/chenggu'],
  ['\u{1F409}', '/shengxiao'],
  ['\u2648', '/xingzuo'],
  ['\u{1F300}', '/qimen'],
  ['\u{1F338}', '/meihua'],
  ['\u{1F3EE}', '/lingqian'],
];
for (const [e, r] of reps) {
  const old = "emoji: '" + e + "', href: '#'";
  const nw = "emoji: '" + e + "', href: '" + r + "'";
  c = c.replace(old, nw);
}
fs.writeFileSync('C:/Users/4513/.openclaw/workspace/jiugong-bagua/src/app/page.tsx', c, 'utf8');
console.log('Remaining # links:', (c.match(/href: '#'/g) || []).length);
